cardano-node/app/cardano-cli.hs

```diff
@@ -286,6 +286,11 @@ parseTxRelatedValues =
             <*> parseTPSRate
                   "tps"
                   "TPS (transaction per second) rate."
+            <*> optional (
+                  parseTxAdditionalSize
+                    "add-tx-size"
+                    "Additional size of transaction (optional value)."
```

Comment:
I think it's extraneous: `(optional value)`.  The help syntax already shows it's optional.

Reply:
Yep, you're right!